### DISCUSSION TITLE
its bug?

### DISCUSSION THREAD

**Discussion by WebAFilippov** [radix-ui/primitives#2608] [Help]
https://www.radix-ui.com/primitives/docs/components/hover-card

![image](https://github.com/radix-ui/primitives/assets/64398851/73918108-225c-4575-b4ee-f45a93ce9e9b)

I see this behavior in Yandex and Google browser. Avatar is not loading

**Comment by benoitgrelard** [comment]
Looks fine everywhere I've looked.
